build/source/engine/mDecisions.f90

```diff
@@ -200,7 +199,7 @@ subroutine mDecisions(err,message)
                   startTime%var(iLookTIME%id),                           & ! day
                   startTime%var(iLookTIME%ih),                           & ! hour
                   startTime%var(iLookTIME%imin),                         & ! minute
-                  dsec,                                                  & ! second
+                  startTime%var(iLookTIME%isec),                         & ! second
```

Comment:
Separate time change from the GRU PR (if possible) and motivate the change from a `real*8` to an integer second (I wonder if that can create problems for the dynamic time stepping, although that may be a different time stamp).